momento-provider/src/main/java/com/google/code/ssm/providers/momento/MomentoClientWrapper.java

```diff
@@ -97,61 +94,40 @@ public void flush() throws CacheException {
     }
 
     @Override
-    public Object get(final String key) throws TimeoutException, CacheException {
-        try {
-            CacheTranscoder cacheTranscoder = getTranscoder();
-            CacheGetResponse response = momentoClient.get(defaultCacheName, key);
-            if (response.byteArray().isPresent()) {
-                byte[] returnedBytes = response.byteArray().get();
-                return cacheTranscoder.decode(new CachedObjectWrapper(
-                        new CachedData(0, returnedBytes, returnedBytes.length)
-                ));
-            }
-            return null;
-        } catch (RuntimeException e) {
-            if (translateException(e)) {
-                throw new CacheException(e);
-            } else if (e.getCause() instanceof TimeoutException) {
-                throw (TimeoutException) e.getCause();
-            }
-            throw e;
+    public Object get(final String key) {
+        CacheTranscoder cacheTranscoder = getTranscoder();
+        CacheGetResponse response = momentoClient.get(defaultCacheName, key);
+        if (response.byteArray().isPresent()) {
+            byte[] returnedBytes = response.byteArray().get();
+            return cacheTranscoder.decode(new CachedObjectWrapper(
+                    new CachedData(0, returnedBytes, returnedBytes.length)
+            ));
         }
+        return null;
     }
 
     @Override
-    public <T> T get(final String key, final CacheTranscoder transcoder) throws CacheException, TimeoutException {
-        Future<CacheGetResponse> f = null;
-        try {
-            Transcoder<T> cacheTranscoder = getTranscoder(transcoder);
-            f = momentoClient.getAsync(defaultCacheName, key);
-            Optional<byte[]> cacheGetResponse = f.get().byteArray();
-            if (cacheGetResponse.isPresent()) {
-                byte[] returnedBytes = cacheGetResponse.get();
-                return cacheTranscoder.decode(new CachedData(0, returnedBytes, returnedBytes.length));
-            }
-            return null;
-        } catch (InterruptedException | ExecutionException e) {
-            cancel(f);
-            throw new CacheException(e);
+    public <T> T get(final String key, final CacheTranscoder transcoder) {
+        Transcoder<T> cacheTranscoder = getTranscoder(transcoder);
+        CacheGetResponse response = momentoClient.get(defaultCacheName, key);
+        Optional<byte[]> cacheGetResponse = response.byteArray();
+        if (cacheGetResponse.isPresent()) {
+            byte[] returnedBytes = cacheGetResponse.get();
+            return cacheTranscoder.decode(new CachedData(0, returnedBytes, returnedBytes.length));
         }
```

Comment:
This comment applies to the other methods, but I'm wondering if we also need to wrap any exceptions we get with a `CacheException`, something like:

```
try {
   ...
} catch (RuntimeException e) {
   throw new CacheException(e);
}
```

Something tells me this library expects `CacheException`